pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/HelixBrokerStarter.java

```diff
@@ -107,7 +108,9 @@ public HelixBrokerStarter(Configuration brokerConf, String clusterName, String z
     _zkServers = zkServer.replaceAll("\\s+", "");
 
     if (brokerHost == null) {
-      brokerHost = NetUtil.getHostAddress();
+      brokerHost =
+        _brokerConf.getBoolean(CommonConstants.Helix.PREFER_HOSTNAME_IN_DEFAULT_INSTANCD_ID_KEY, false) ? NetUtil
```

Comment:
Typo Instance can we shorten the constant. SET_INSTANCEID_TO_HOSTNAME

Reply:
done